scripts/git-checks.sh

```diff
@@ -52,8 +52,9 @@ done
 # For example, no carriage returns or line feeds should be allowed in filenames.
 export LC_ALL=C
 for rev in $(git rev-list $oldrev..$newrev); do
+	# ignore the developer website directory
 	names=$(git diff-tree $rev -r --root --no-commit-id --name-only \
-		--diff-filter=AR)
+		--diff-filter=AR) | grep -v '^developers.libra.org/'
```

Comment:
We shouldn't ignore the website here. Which checks do you want disabled? If its whitespace ones then you should add a .gitattributes file turning off whitespace checks. See as an example https://github.com/libra/libra/blob/master/.github/actions/hyperjump-comment/.gitattributes

Reply:
This is an old commit; I used a different exclusion mechanism to ignore filenames and whitespace. Those are the only two places I did the exclusion.